package/kedro_viz/integrations/deployment/azure_deployer.py

```diff
@@ -0,0 +1,84 @@
+"""`kedro_viz.integrations.deployment.azure_deployer` defines
+deployment class for Azure Blob Storage"""
+import glob
+import logging
+import mimetypes
+from pathlib import Path
+
+from kedro_viz.integrations.deployment.base_deployer import BaseDeployer
+
+try:
+    from azure.storage.blob import ContentSettings
+except ImportError:  # pragma: no cover
+    pass
+
+import fsspec
+
+from kedro_viz import __version__
+
+_AZ_PROTOCOL = "abfs"
```

Comment:
Can we reused logic from kedro? I don't think `abfs` is the only protocols supported by Azure. See https://github.com/fsspec/adlfs

Reply:
Here we are using Gen2 filesystem to build the remote path (we will mention this in our docs that azure deployment is supported with gen2 filesystem). Could you please point me to what reuse logic from kedro you are referring to ? Thank you

Reply:
https://github.com/kedro-org/kedro/blob/80ad182a220d691c77efc05cc4e2b36dcecb6ac7/kedro/io/core.py#L711
95% identical to `fsspec` to handle some extra edge cases. If this only support `abfs` it is fine too, but worth mentioning in the docs.